easypermissions/src/main/java/pub/devrel/easypermissions/RationaleAppCompatDialogFragment.java

```diff
@@ -0,0 +1,114 @@
+package pub.devrel.easypermissions;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.StringRes;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+
+import java.util.Arrays;
+
+import static pub.devrel.easypermissions.EasyPermissions.executePermissionsRequest;
+
+
+public class RationaleAppCompatDialogFragment extends DialogFragment implements Dialog.OnClickListener {
```

Comment:
Add class-level JavaDoc comments to each of these Fragment classes explaining their slightly different purposes.